src/NodaTime.Test/XmlSchemaTest.cs

```diff
@@ -0,0 +1,298 @@
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+using ApprovalTests;
+using ApprovalTests.Reporters;
+using NodaTime.Calendars;
+using NUnit.Framework;
+using static NodaTime.XmlSchemaDefinition;
+
+namespace NodaTime.Test
+{
+    [UseReporter(typeof(DiffReporter))]
+    public class XmlSchemaTest
+    {
+        [Test]
+        public void XmlSchema()
+        {
+            var xml = GetXmlSchemaString(NodaTimeXmlSchema);
+            Approvals.VerifyWithExtension(xml, "xml");
+        }
+
+        [Test]
+        public void XmlSchemaExporter()
+        {
+            var namespaceManager = new XmlNamespaceManager(new NameTable());
+            namespaceManager.AddNamespace(NodaTimeXmlNamespace.Name, NodaTimeXmlNamespace.Namespace);
+            var xmlSchema = GetXmlSchema<NodaTimeObject>();
+            var xml = GetXmlSchemaString(xmlSchema, namespaceManager);
+            Approvals.VerifyWithExtension(xml, "xml");
+        }
+
+        [Test]
+        public void ValidXmlDocumentWithAllTypes()
+        {
+            var xml = "<?xml version=\"1.0\"?>" +
```

Comment:
I think a verbatim string literal would be clearer here.

Reply:
You're right, I will fix it and force push.